### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by jchodera** [choderalab/openmoltools#144]
Perhaps it's time to consider writing some documentation for this toolkit?  It might be a good strategy to write docs alongside the actual code, especially since this is a distributed development project.

If there is interest, I can set up a documentation skeleton and automate pushing docs to [`readthedocs`](http://pymbar.readthedocs.org), much like [`pymbar` does](http://pymbar.readthedocs.org/en/latest/).

**Comment by davidlmobley** [comment]
Yes. That would be great. It would also be awesome if you could tell me how
this works, at the same time. (i.e. I know how to write the text of
documentation, but I don't know how to get it into a useful format, nor how
the automation of this works, etc.)

On Fri, Jul 3, 2015 at 10:08 AM, John Chodera <notifications@github.com>
wrote:

> Perhaps it's time to consider writing some documentation for this toolkit?
> It might be a good strategy to write docs alongside the actual code,
> especially since this is a distributed development project.
>
> If there is interest, I can set up a documentation skeleton and automate
> pushing docs to readthedocs <http://pymbar.readthedocs.org>, much like
> pymbar does <http://pymbar.readthedocs.org/en/latest/>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/choderalab/openmoltools/issues/144>.
>



-- 
David Mobley
Associate Professor
Department of Pharmaceutical Sciences
Department of Chemistry
3134B Natural Sciences I
University of California, Irvine
Irvine, CA 92697
dmobley@uci.edu
work (949) 824-6383
cell (949) 385-2436
